cime/src/components/data_comps_mct/dlnd/cime_config/namelist_definition_dlnd.xml

```diff
@@ -138,6 +154,8 @@
     <values>
       <value stream="sno.cplhist">$DLND_CPLHIST_CASE.cpl.hl2x1yr_glc.%y-01-01.nc</value>
       <value stream="clm.nldas">mosart_nldas_hist_1975.clm2.h1.1975-01-01-00000.nc</value>
+      <value stream="lnd.qian">QIAN.daily.1948-2004.nc</value>
+      <value stream="lnd.gpcc">GPCC.daily.nc</value>
```

Comment:
Do we need to upload these two nc files to the server or they are already there?

Reply:
@hydrotian, @apcraig, I missed this comment and have merged the PR to next but I'm going to revert it. Where are these files on Compy and do we need these new files?

Reply:
Both data files are on compy's inputdata area,

/compyfs/inputdata/lnd/dlnd7/hcru_hcru/

I do not know whether these files have been added to the inputdata server or whether they should be.

Reply:
@bishtgautam I guess you may not need them to pass the tests, but these are default runoff forcing data for the new usrdat functions.